crypto/bio/bf_stdinbuff.c

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/*
+ * ftell and seek do not work for a file bio that uses stdio.
+ * This filter bio's main purpose is to sit on top of a stdio file bio
+ * so that tell and seek work. It does this by caching ALL file data into
+ * a resizable memory buffer.
+ */
```

Comment:
Is there a reason it can't be used on top of POSIX I/O (file descriptor) BIOs?

Reply:
No it could be used for that also.. What would you like it to say here.

Reply:
I would like it to say that it's designed to be placed on top of any BIO that doesn't support seek or tell, and mention stdio as a mere example